packages/common/src/project/readers/ipfs-reader.ts

```diff
@@ -49,8 +49,8 @@ export class IPFSReader implements Reader {
       }
       return await this.cache[fileName];
     } catch (e) {
-      console.error(`Failed to fetch project from IPFS: ${fileName}`, e);
-      throw new Error(`Failed to fetch project from IPFS: ${fileName}`, {cause: e});
+      console.error(`Failed to fetch file from IPFS: ${fileName}`, e);
+      throw new Error(`Failed to fetch file from IPFS: ${fileName}`, {cause: e});
```

Comment:
I noticed that the cause doesn't get logged if theres an error. Can you see if theres a way to change that?